core/services/eth/log_broadcaster.go

```diff
@@ -379,3 +381,12 @@ func (l *decodingLogListener) HandleLog(log interface{}, err error) {
 
 	l.LogListener.HandleLog(decodedLog, nil)
 }
+
+func appendLogChannel(oldLogChanel, newLogChanel chan eth.Log) chan eth.Log {
+	go func() {
+		for rawLog := range oldLogChanel {
+			newLogChanel <- rawLog
```

Comment:
mispelling of channel

Reply:
uhg. I'm switching all my variable names to spanish, where the spelling actually consistently matches the phonetics :facepalm: